src/test/java/com/sap/sgs/phosphor/fosstars/model/score/oss/ProjectActivityScoreTest.java

```diff
@@ -104,15 +104,20 @@ public void commitsWithoutContributors() {
     PROJECT_ACTIVITY.calculate(values(1, 0));
   }
 
-  @Test(expected = IllegalArgumentException.class)
+  @Test
```

Comment:
I think that this case is still invalid. The score should throw an exception here. Or, can you come up with an example which proves the opposite?

Reply:
This might be a special case, but you are right. Contributor without any commit also doesn't add to our usecase.